opds.py

```diff
@@ -123,13 +123,15 @@ def slugify_feed_title(cls, feed_title):
         slug = re.sub(' {2,}', ' ', slug)
         return unicode('-'.join(slug.split(' ')))
 
-    def __init__(self, base_url, lane=None, include_search=None, prefix=None):
+    def __init__(self, base_url, lane=None, prefix=None, include_search=None,
+                 license_link=None, suppressed_covers=None):
         if not base_url.endswith('/'):
```

Comment:
is the change of order for the prefix and include_search params intentional?  (are there any scripts calling them that need cleaning?)